setup.py

```diff
@@ -363,6 +378,9 @@ class BuildExt(build_ext):
         linux_opts = ["-O3"]
         c_opts["unix"] += linux_opts
         l_opts["unix"] += linux_opts
+    elif sys.platform == "win32":
+        windows_opts = ["/fsanitize=address"]
+        c_opts["msvc"] += windows_opts
```

Comment:
Looks unnecessary to use `windows_opts` temp variable, can assign `c_opts["msvc"]` directly